website/docs/addon_resolve_artist.md

```diff
@@ -233,13 +235,14 @@ There are two ways to create a publishable editorial package: by creating a new
 
 ### Create/Convert a Timeline from Media Pool into an Editorial Package
 
-To create an editorial package, select a timeline in the media pool (1) and use the Create tool (2) to mark the timeline with Editorial Package (3) publishable product data. Keep _`Use selection`_ (5) active and choose _`Editorial Package`_ (3) as the product type. You can also use a Variant (4) to differentiate between different versions of timelines within the same target context. Hit Create (6).
+To create an editorial package, select a timeline in the media pool (1) and use the Create tool (2).
```

Comment:
Is it worth it here to clarify we're referring to the AYON Create?
```suggestion
To create an editorial package, select a timeline in the media pool (1) and use the AYON Create tool (2).
```

Reply:
Looks good to me !